frontend/src/pages/MessagingQueues/MQDetails/MQTables/getPartitionLatencyOverview.ts

```diff
@@ -0,0 +1,34 @@
+import axios from 'api';
+import { ErrorResponseHandler } from 'api/ErrorResponseHandler';
+import { AxiosError } from 'axios';
+import { SOMETHING_WENT_WRONG } from 'constants/api';
+import { ErrorResponse, SuccessResponse } from 'types/api';
+
+import {
+	MessagingQueueServicePayload,
+	MessagingQueuesPayloadProps,
+} from './getConsumerLagDetails';
+
+export const getPartitionLatencyOverview = async (
+	props: Omit<MessagingQueueServicePayload, 'detailType' | 'variables'>,
+): Promise<
+	SuccessResponse<MessagingQueuesPayloadProps['payload']> | ErrorResponse
+> => {
+	try {
+		const response = await axios.post(
+			`/messaging-queues/kafka/partition-latency/overview`,
```

Comment:
also why are the APIs part of the `pages` folder. we have separate types and api folders. move them there.